docs/source/cdk/aws-setup.rst

```diff
@@ -0,0 +1,43 @@
+AWS Setup
+=========
+Download/requirements
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
=========

Download/requirements
~~~~~~~~~~~~~~~~~~~~~
```
Extra linebreak between the sections, and it is nice to make the underlines exactly the same length even though the spec does allow longer.

I would have said an extra linebreak after even the minor section underlines, but it doesn't look like @maxinelasp did that in imap-processing. I would follow her lead for consistency, which I think did include the linebreak between the primary page header.